frontend/pages/admin/OrgSettingsPage/cards/Agents/Agents.tsx

```diff
@@ -58,7 +66,7 @@ const Agents = ({
     evt.preventDefault();
 
     // Formatting of API not UI and allows empty agent options
-    const formDataToSubmit = agentOptions
+    const formDataToSubmit: any = agentOptions
       ? {
           agent_options: yaml.load(agentOptions),
```

Comment:
For future iterations: `yaml.load` returns  `object` | `string` | `number` | `null`| `undefined`. I wonder if we can surmise anything about the actual possible contents of the `yaml` from `IConfig.agent_options` (what can that `yaml` actually possibly contain?), and cast this return value appropriately, so that we can say something more specific about `formDataToSubmit` than `any`. Something like:

`type AgentOptionsParsed = { config: {} } | <whatever yaml.load(agentOptions) can actually look like>;

interface IAgentOptionsFormDataToSubmit {
  agent_options: AgentOptionsParsed;
}

formDataToSubmit: IAgentOptionsFormDataToSubmit`...

Reply:
yass feels like some good spring cleaning!